admin/app/components/solidus_admin/orders/show/component.html.erb

```diff
@@ -8,5 +8,68 @@
     <% end %>
   <% end %>
 
-  <%= render component('orders/cart').new(order: @order) %>
+  <%= page_with_sidebar do %>
+    <%= page_with_sidebar_main do %>
+      <%= render component("orders/cart").new(order: @order) %>
+    <% end %>
+
+    <%= page_with_sidebar_aside do %>
+      <%= render component('ui/panel').new(title: panel_title_with_more_links(t(".customer"), [
+        link_to("Edit order email", "#", class: "p-2 hover:bg-gray-25 rounded-sm text-black"),
+        link_to("Edit shipping address", "#", class: "p-2 hover:bg-gray-25 rounded-sm text-black"),
+        link_to("Edit billing address", "#", class: "p-2 hover:bg-gray-25 rounded-sm text-black"),
+        link_to("Remove customer", "#", 'data-turbo-method': :delete, class: "p-2 hover:bg-gray-25 rounded-sm text-red-500"),
```

Comment:
Should we also define the link_to bodies in component.yml?

Reply:
Done thanks!